packages/chain-adapters/src/evm/EvmBaseAdapter.ts

```diff
@@ -665,6 +665,9 @@ export abstract class EvmBaseAdapter<T extends EvmChainId> implements IChainAdap
 
   async getFeeData(input: GetFeeDataInput<T>): Promise<FeeDataEstimate<T>> {
     try {
+      if (isHex(input.value)) {
+        throw new Error('value must be a stringified BN, not an hex string')
+      }
```

Comment:
saves a request, but returns essentially the same error as the failed request no? would personally omit and I will add an input validation ticket in unchained to handle number string vs hex string better

Reply:
https://github.com/shapeshift/unchained/issues/1124

Reply:
Yeah it's already handled by unchained, if we feel like this is useless, happy to revert this bit! 
Eventually with proper nominal type for a numeric string (*not*  an hex string) we should be able to avoid this at type-level, more of a bandaid.

Reply:
my vote is revert this given the other changes going in. not a huge deal, but extra and redundant in the end goal state.

Reply:
b56f624c50d73bee868f7b186664387de2905fbc